src/main/resources/db/changelog/changes/create-transaction-table.xml

```diff
@@ -13,6 +12,14 @@
     </sql>
   </changeSet>
 
+  <!-- Create StatusEnum ENUM -->
```

Comment:
As the code is already deployed in testing environment, Please add this in a new xml file and add it to changelog file.